utils.js

```diff
@@ -0,0 +1,14 @@
+const db = require("./db/connection");
+
+exports.doesArticleExist = (article_id) => {
+  return db
+    .query(
+      `
+    SELECT *
+    FROM articles
+    WHERE article_id = $1
+    `,
+      [article_id],
+    )
+    .then(({ rows: articles }) => articles.length > 0);
+};
```

Comment:
Very good approach to have a separate function for this, there is some question in my mind of whether this would be a util function or a model, because it is interacting with the database to validate data - which is exactly the responsibility of a model function 🤔